src/docs/shared/FgUsage.vue

```diff
@@ -1,38 +1,38 @@
 <template>
   <div>
-    <header-large>
-      Usage
-    </header-large>
+    <fg-header-large>
+      FgUsage
```

Comment:
This text shouldn't be prefixed:
```suggestion
      Usage
```